flutter-boilerplate/app_feup/lib/view/Pages/LoginPageView.dart

```diff
@@ -0,0 +1,172 @@
+                    ])),
+            createLogInButton(),
+            Padding(padding: EdgeInsets.only(bottom: 50)),
+            Expanded(flex: 1, child: createNoteLabel()),
+          ],
+        ),
+      )),
+    );
+  }
+
+  Widget createTitle() {
+    return Text(
+      "APP\nFEUP",
+      textAlign: TextAlign.center,
+      style: TextStyle(
+          color: Colors.white, fontSize: 50.0, fontWeight: FontWeight.w400),
+    );
+  }
+
+  Widget createUsernameInput(BuildContext context) {
+    return TextFormField(
+      style: new TextStyle(color: Colors.white),
+      keyboardType: TextInputType.number,
+      autofocus: false,
+      controller: usernameController,
+      focusNode: usernameFocus,
+      onFieldSubmitted: (term) {
+        usernameFocus.unfocus();
+        FocusScope.of(context).requestFocus(passwordFocus);
+      },
+      textInputAction: TextInputAction.next,
+      textAlign: TextAlign.center,
+      decoration: textFieldDecoration('student nr'),
+      validator: (String value) =>
+          value.isEmpty ? 'Student number can\'t be empty' : null,
+    );
+  }
+
+  Widget createPasswordInput() {
+    return TextFormField(
+        style: new TextStyle(color: Colors.white),
+        autofocus: false,
+        controller: passwordController,
+        focusNode: passwordFocus,
+        onFieldSubmitted: (term) {
+          passwordFocus.unfocus();
+          submitForm();
+        },
+        textInputAction: TextInputAction.done,
+        obscureText: true,
+        textAlign: TextAlign.center,
+        decoration: textFieldDecoration('password'),
+        validator: (String value) =>
+            value.isEmpty ? 'Password can\'t be empty' : null);
+  }
+
+  Widget createSaveDataCheckBox() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: <Widget>[
+        Text("Manter sessão iniciada",
+            textAlign: TextAlign.center,
+            style: TextStyle(
+                color: Colors.white,
+                fontSize: 13.0,
+                fontWeight: FontWeight.w300)),
+        Checkbox(value: checkboxValue, onChanged: setCheckboxValue)
+      ],
+    );
+  }
+
+  Widget createLogInButton() {
+    return new SizedBox(
+      width: 150,
+      height: 50,
+      child: RaisedButton(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(25),
+        ),
+        onPressed: submitForm,
+        padding: EdgeInsets.all(12),
+        color: Colors.white,
+        child: Text('Log In',
+            style: TextStyle(
+                color: primaryColor, fontWeight: FontWeight.w400, fontSize: 18),
+            textAlign: TextAlign.center),
+      ),
+    );
+  }
+
+  Widget createNoteLabel() {
+    return StoreConnector<AppState, String>(
+        converter: (store) => store.state.content['loginMessage'],
+        builder: (context, message) {
+          return Text(
+            message ?? 'No message',
```

Comment:
Instead of 'No message' it would maybe be better not to show anything until the user tries to log in (until there is a message to show)?